lib/winston/create-logger.js

```diff
@@ -76,6 +76,15 @@ module.exports = function (opts = {}) {
       if (args.length === 1) {
         const [msg] = args;
         const info = msg && msg.message && msg || { message: msg };
+
+        // let info;
+        // // Handling of Error instances has been moved here from Child logger objects so it can apply at all times.
+        // if (msg instanceof Error) {
+        //   info = { message: msg.message, stack: msg.stack };
+        // } else {
+        //   info = msg && msg.message && msg || { message: msg };
+        // }
+
```

Comment:
If this code is all commented out, should it be deleted before merge?

Reply:
Most likely. 😅

Reply:
It doesn't look like it got removed!

Reply:
Ok so I've since refreshed my memory. This was moved from the child [override of the `write` function here](https://github.com/winstonjs/winston/blob/038ae23b00136c1e01c18a137ed0339b720bbb31/lib/winston/logger.js#L48-L69). Per comments by @indexzero it should be removed. I don't think it necessary to retain it anywhere and all tests are passing without it. Thoughts?